docs/docs/plugin_README/offload_vio.md

```diff
@@ -14,15 +14,22 @@ and the device running in a different terminal. **Each terminal must be running
 repositories** (clone ILLIXR twice to different locations; running both the server and device from the same 
 ILLIR repository will cause lock issues). 
 
-Start ILLIXR using the runner with configs/offload-server.yaml
-for one terminal and configs/offload-server.yaml on the other terminal. The default configuration within the
-config files will run OpenVINS on the server and will feed it with the EuRoC dataset from the device.
+Start ILLIXR using
+``` bash
+main.opt.exe --yaml=profiles/offload-server.yaml --data=<> --demo_data=<>
+```
+for one terminal and 
+``` bash
+main.opt.exe --yaml=profiles/offload-device.yaml --data=<> --demo_data=<>
```

Comment:
Have we tested this?

Reply:
I tested it on my laptop and it works fine. Please try on your side if you have time.